packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -795,6 +798,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return this.context.attachState;
     }
 
+    public get loader(): ILoader {
```

Comment:
I'm not sure it's good idea to expose loader, given all the latest discussions about ability to create Container directly (3P related discussions). Can we avoid it?

Reply:
I wondered about that. I haven't found another way to access the loader I need, but I'll keep looking.

Reply:
I added a ContainerRuntime method that will return a request-summarizer-function, so the loader doesn't need to be exposed.